root_orchestrator/cloud-scheduler/requirements.txt

```diff
@@ -5,4 +5,7 @@ click~=8.1.7
 redis~=5.0.1
 requests~=2.27.1
 werkzeug==2.0.3
-git+https://github.com/oakestra/oakestra.git@develop#subdirectory=libraries/resource_abstractor_client
+#git+https://github.com/oakestra/oakestra.git@develop#subdirectory=libraries/resource_abstractor_client
+git+https://github.com/oakestra/oakestra.git@service-status-enhancement#subdirectory=libraries/resource_abstractor_client
+git+https://github.com/oakestra/oakestra.git@service-status-enhancement#subdirectory=libraries/oakestra_utils_library
+icecream
```

Comment:
do we need icecream library?

Reply:
Good catch. I use it for debugging. So I will remove it before merging things. TODO @Malyuk-A